server/controllers/LikeController.js

```diff
@@ -0,0 +1,128 @@
+import db from '../database/models';
+
+const { Article, ArticleLike, User } = db;
+
+
+export default class LikeController {
+  static likeArticle(req, res, next) {
+    const { slug } = req.params;
+    const { decoded: user } = req.body;
+    Article.findOne({
+      where: {
+        slug,
+      },
+      include: [
+        {
+          model: User,
+          as: 'author',
+          attributes: [
+            'firstName', 'lastName',
+            'username', 'image', 'bio',
+            'id',
+          ],
+        },
+      ],
+      attributes: ['id', 'title', 'slug'],
+    }).then((article) => {
+      if (article) {
+        const likeObj = {
+          userId: user.id,
+          articleId: article.dataValues.id,
+        };
+        ArticleLike.findOrCreate({
+          where: likeObj,
+          defaults: likeObj,
+          include: [
+            {
+              model: Article,
+              as: 'article',
+            },
+
+          ],
+          attributes: [
+            'userId', 'articleId'],
+        })
+          .then(([, created]) => {
+            res.status(created ? 201 : 200).json({
+              status: 'success',
+              data: article.dataValues,
+              message: created
+                ? 'Successfully liked the article'
+                : 'You had already liked this article',
+            });
+          });
+      } else {
+        res.status(404).json({
+          status: 'error',
+          error: {
+            message: 'The slug you specified was not found',
+            number: 404,
```

Comment:
Why do you have the key/value `number: 404` when you had already indicated `404` as the status code?